src/Services/SearchService.php

```diff
@@ -434,7 +435,9 @@ private function getProductDetailUrl(Response $response)
 
     private function shouldExportWithVariationId(int $variationId): bool
     {
-        $showPleaseSelect = $this->pluginInfoService->isOptionShowPleaseSelectEnabled('Ceres');
+        /** @var TemplateConfigService $templateConfigService */
+        $templateConfigService = pluginApp(TemplateConfigService::class);
+        $showPleaseSelect = $templateConfigService->getBoolean('item.show_please_select');
```

Comment:
I still think it makes sense to have the method, as it hides the internal naming schema and exposes plugin configuration in an easily obtainable way. The `PluginInfoService` could also have the `TemplateConfigService` available inside the constructor to load it automatically via dependency injection.